RRS/staff/views.py

```diff
@@ -19,7 +22,17 @@ def create_reservation(request):
 
 def edit_restaurant(request):
     
-    context = {}
+    # TODO(noelsison2): Get current user's associated restaurant
+    restaurant = get_object_or_404(Restaurant, pk=1)
```

Comment:
Yup, we haven't made user roles and associated restaurant with users yet. I made issue #20 for this.